Makefile

```diff
@@ -56,24 +56,29 @@ lint:
 	rm requirements.commit
 
 functional-tests-exitfirst:
-	pytest --exitfirst --gherkin-terminal-reporter tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
+ifeq ($(SLOW),)
+	echo "Running fast tests"
+	pytest --exitfirst --splits 3 --group $(PYTEST_GROUP) --store-durations --durations-path=./.test_durations.$(PYTEST_GROUP) --deselect=tests/functional/targets/test_performance.py::test_api_requester_multiple_request_and_targets[50-50-600] tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
```

Comment:
What's the benefit of providing the `PYTEST_GROUP` argument?

Reply:
It's necessary when using the `pytest-split` plugin:

So if you choose to split your tests into 3 groups, you will have something like:
```shell
pytest --splits 3 --group 1
pytest --splits 3 --group 2
pytest --splits 3 --group 3
```

See https://jerry-git.github.io/pytest-split/#usage

Reply:
Isn't it better to use `least_duration` instead of an absolute order? It's more effective over time